src/activitypub/fediverse-bridge.ts

```diff
@@ -106,7 +121,7 @@ export class FediverseBridge {
             actor: post.author.apId,
             object: fedifyObject,
             to: PUBLIC_COLLECTION,
-            cc: post.author.apFollowers,
+            ccs: ccs,
```

Comment:
We want to add cc in both `fedifyObject` and `createActivity`. Sample [Create activity from Mastodon](https://browser.pub/https%3A%2F%2Fmastodon.social%2Fusers%2Fprincivershwal%2Fstatuses%2F114506263585764479%2Factivity)